docs/src/main/asciidoc/security-getting-started.adoc

```diff
@@ -3,70 +3,73 @@ This guide is maintained in the main Quarkus repository
+== How to complete this tutorial
 
-== Solution
+You can start from scratch and complete each of the numbered steps in sequence or you can bypass the basic setup steps that are already familiar to you:
 
-We recommend that you follow the instructions in the next sections and create the application step by step.
-However, you can go right to the completed example.
-
-Clone the Git repository: `git clone {quickstarts-clone-url}`, or download an {quickstarts-archive-url}[archive].
+* To start from scratch, go to *Create a Maven project with the `security-jpa` extension*.
 
+* To skip the basics and go straight to the end solution, either:
+** Clone the Git repository: `git clone {quickstarts-clone-url}`
+** Download the {quickstarts-archive-url}[archive]
+[TIP]
+====
 The solution is located in the `security-jpa-quickstart` {quickstarts-tree-url}/security-jpa-quickstart[directory].
+====
 
-== Creating the Maven Project
+:sectnums:
+:sectnumlevels: 3
 
-First, we need a new project. Create a new project with the following command:
+== Create a Maven project with the `security-jpa` extension
+
+For Quarkus security to be able to map your security source to JPA entities, ensure that the Maven project that is used in this tutorial includes the `security-jpa` extension.
+You can either create a new Maven project with the `security-jpa` extension or you can add the extension to an existing Maven project.
+
+* To create the Maven project, use the following command:
 
 :create-app-artifact-id: security-jpa-quickstart
 :create-app-extensions: security-jpa,jdbc-postgresql,resteasy-reactive,hibernate-orm-panache
 include::{includes}/devtools/create-app.adoc[]
-
-This command generates a Maven project, importing the `security-jpa` extension
-which allows you to map your security source to JPA entities.
-
 [NOTE]
 ====
 xref:hibernate-orm-panache.adoc[Hibernate ORM with Panache] is used to store your user identities but you can also use xref:hibernate-orm.adoc[Hibernate ORM].
-
-Don't forget to add the database connector library of choice. Here we are using PostgreSQL as identity store.
+You must also add your preferred database connector library.
+The instructions in this example tutorial use a PostgreSQL database for the identity store.
 ====
-
-If you already have your Quarkus project configured, you can add the `security-jpa` extension
-to your project by running the following command in your project base directory:
+* To add the `security-jpa` extension to an existing Maven project, run the following command from your project base directory:
 
 :add-extension-extensions: security-jpa
 include::{includes}/devtools/extension-add.adoc[]
 
-This will add the following to your build file:
+=== Verification
+
+When you run either command, the following XML is added to your build file:
```

Comment:
I added 'either command' here to the original line 69. Is this correct?

I wasn't quite sure if the verification information applied to a) creating a new Maven project with security-jpa` extension AND  b) Adding security-jpa` extension to an existing Maven, or just the latter.

Reply:
`either` should be fine, thanks